src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryServiceTest.java

```diff
@@ -579,6 +579,48 @@ void shouldReturnRequestContextAvailableTasksOnly() {
             assertEquals("Hearing work", taskResourceList.getTasks().get(0).getWorkTypeLabel());
         }
 
+        @Test
+        void shouldReturnRequestContextAvailableTasksOnlyAndIgnoreSearchParameterAvailableTasksOnly() {
```

Comment:
Minor point but would remove reference to `Only` when referring to the request context where possible.

Reply:
Changed to shouldReturnRequestContextAvailableTasksAndIgnoreSearchParameterAvailableTasksOnly, and some other places as well.